standalone/ansible/group_vars/webservers.yml

```diff
@@ -1,3 +1,4 @@
+nginx_port: 8008  # Use 8008 (HTTP Alternate) to avoid conflict with HAProxy in case deployment is on a single box
```

Comment:
This will still work when deployed on separate boxes, right?

Reply:
Yes, this works on separate boxes too.